tests/unit/net/cookie.rs

```diff
@@ -132,3 +133,160 @@ fn test_sort_order() {
     assert!(CookieStorage::cookie_comparator(&a_prime, &a) == Ordering::Greater);
     assert!(CookieStorage::cookie_comparator(&a, &a) == Ordering::Equal);
 }
+
+
+fn run(set_location: &str, set_cookies: &[String], final_location: &str) -> String {
+    let mut storage = CookieStorage::new();
+    let url = ServoUrl::parse(set_location).unwrap();
+    let source = CookieSource::HTTP;
+
+    // Add all cookies to the store
+    for str_cookie in set_cookies {
+        let bytes = str_cookie.to_string().into_bytes();
+        let header = Header::parse_header(&[bytes]);
```

Comment:
Instead of having `if let Ok(..)` on the next line (which would skip the code if the header did not parse correctly), we should use `let header = Header::parse_header(&[bytes]).unwrap();`, so the test will fail if the header does not parse correctly.